Gruntfile.js

```diff
@@ -138,6 +138,8 @@ module.exports = function Gruntfile( grunt ) {
 			manifest: {
 				options: {
 					patterns: [
+						// The trailing slash here is also used in the compress.webextSource Grunt
```

Comment:
You mean the trailing `.0`?

Reply:
The phrase "trailing slash" is too deep in my fingers' muscle memory, obviously!

Fixed. :-)